src/java.com/homework6/Chamomile.java

```diff
@@ -0,0 +1,14 @@
+package homework6;
```

Comment:
homework6 is not bad. Let's make the root package for example. com.flower.kiev.
And put all flower-classes into the package entity or domain.